### ISSUE TITLE
Incorrect genearate Links for Headers

### ISSUE THREAD

**Issue by kvandake** [davidcelis/api-pagination#118]

**Comment by davidcelis** [comment]
Yeah, it looks like both Rails and grape just hardcode `page` as the parameter name despite configuration.

To be honest, I'm not sure how to solve this given how the configuration class currently works. The configuration options for the `page` and `per_page` parameter names take a block, and the block can contain anything. There can be any sort of nesting within the `params` hash, and I can't see any way of actually retrieving the names themselves so that I can just set the new param.
